tools/generators/rc-caret/index.ts

```diff
@@ -0,0 +1,106 @@
+import { Tree, updateJson, getProjects, formatFiles } from '@nrwl/devkit';
+import * as semver from 'semver';
+import { VersionBumpGeneratorSchema } from './schema';
+import { getProjectConfig, isPackageVersionConverged, printUserLogs, UserLog } from '../../utils';
+import { PackageJson } from '../../types';
+
+export default async function (host: Tree, schema: VersionBumpGeneratorSchema) {
```

Comment:
nit: I believe we've adopted using the variable name `tree` instead of `host` for better readability (this is also what's used in our boilerplate workspace generator)
```suggestion
export default async function (tree: Tree, schema: VersionBumpGeneratorSchema) {
```